src/IdeaStatiCa.Plugin/BIMPluginHosting.cs

```diff
@@ -156,8 +194,24 @@ private void SelfServiceHost_Opened(object sender, EventArgs e)
 			}
 		}
 
+		private void SelfServiceHost_UnknownMessageReceived(object sender, UnknownMessageReceivedEventArgs e)
+		{
+			ideaLogger.LogDebug($"SelfServiceHost_UnknownMessageReceived service '{ServiceBaseAddress}'");
+		}
+
+		private void SelfServiceHost_Opening(object sender, EventArgs e)
+		{
+			ideaLogger.LogDebug($"SelfServiceHost_Opening service '{ServiceBaseAddress}'");
+		}
+
+		private void SelfServiceHost_Faulted(object sender, EventArgs e)
+		{
+			ideaLogger.LogError($"SelfServiceHost_Faulted service '{ServiceBaseAddress}'", new Exception());
```

Comment:
Please log the error details, if possible.

```suggestion
			ideaLogger.LogError($"SelfServiceHost_Faulted service '{ServiceBaseAddress}'");
```

Reply:
error details are not available here. it is wcf event handler